src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -356,8 +356,8 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       const removesDefaultHiding = category === WearableCategory.UPPER_BODY ? [BodyPartCategory.HANDS] : []
       data = {
         ...pristineItem.data,
-        replaces: [],
-        hides: [],
+        replaces: pristineItem.data.replaces ?? [],
```

Comment:
removing these lines could work too?  in the previous line, it's already getting the `...pristineItem.data`

Reply:
They actually, do! I'll remove them